client/dist/style.css

```
@@ -0,0 +1,96 @@
+ul{
+  padding: 0;
+  list-style: none;
+}
+
+#app {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  height: 252px;
+  width: 300px;
+}
+
+.rel_songStats {
+  color: #999;
+  padding: 0px;
+}
+.rel_songStats li {
+  display: inline;
+}
+
+.rel_artworkPlaceholder {
```

This and the one below can maybe be combined together?
```suggestion
.rel_artworkPlaceholder, .rel_albumImg {
```